apps/gitness/src/pages/repo-sandbox/repo-sandbox-summary.tsx

```diff
@@ -179,10 +195,23 @@ export const RepoSandboxSummaryList: React.FC = () => {
               </ListActions.Left>
               <ListActions.Right>
                 <ButtonGroup.Root>
-                  <Button variant="outline">
-                    Add file&nbsp;&nbsp;
-                    <Icon name="chevron-down" size={11} className="chevron-down" />
-                  </Button>
+                  <DropdownMenu>
+                    <DropdownMenuTrigger asChild>
+                      <Button variant="outline">
+                        Add file&nbsp;&nbsp;
+                        <Icon name="chevron-down" size={11} className="chevron-down" />
+                      </Button>
+                    </DropdownMenuTrigger>
+                    <DropdownMenuContent align="start">
+                      <DropdownMenuItem
+                        key={'create-file'}
+                        onClick={() => {
+                          navigate(`/${spaceId}/repos/${repoId}/code/new/${gitRef || selectedBranch}/~/`)
+                        }}>
```

Comment:
if we are using the path params only here we could possibly pass a relative path 

```
navigate('../code/new/${gitRef || selectedBranch}/~/')
```

instead of passing the entire path using path params. 

https://reactrouter.com/en/main/hooks/use-navigate#optionsrelative

Reply:
wont be able to use relative until all pages are migrated and we don't use sandbox prefix anymore